inst/tests/nafill.Rraw

```diff
@@ -149,8 +149,15 @@ test(4.20, colnamesInt(dt, integer()), integer())
 test(4.21, colnamesInt(dt, NULL), seq_along(dt))
 test(4.22, colnamesInt("asd", 1), error="must be data.table compatible")
 test(4.23, colnamesInt(dt, 1, check_dups="a"), error="check_dups")
+test(4.24, colnamesInt(dt, c("a", "e"), skip_absent=TRUE), c(1L,0L))
+test(4.25, colnamesInt(dt, c(1L, 4L), skip_absent=TRUE), c(1L,4L))
+test(4.26, colnamesInt(dt, c(1, 4), skip_absent=TRUE), c(1L,4L))
+test(4.27, colnamesInt(dt, c("a", NA), skip_absent=TRUE), c(1L,0L))
```

Comment:
we're missing a test of numeric input larger than `ncol` e.g. `c(1L, 1000L)` here

Reply:
done 👍